vms/proposervm/scheduler/scheduler.go

```diff
@@ -101,9 +104,14 @@ waitloop:
 }
 
 func (s *scheduler) SetBuildBlockTime(t time.Time) {
+	if s.shutdown.Get() {
```

Comment:
avoid risk of sending time over a closed channel when Close is concurrently called